src/api/metadata/types.ts

```diff
@@ -1,30 +1,15 @@
-type Metadata = {
-  version: string
-}
-
-export type ReferralMetadata = Metadata & {
-  address: string
-}
+import { createAppDataDoc, createQuoteMetadata, createReferrerMetadata } from '@cowprotocol/app-data'
 
-export type QuoteMetadata = Metadata & {
-  slippageBips: string
-}
+export { AnyAppDataDocVersion, LatestAppDataDocVersion } from '@cowprotocol/app-data'
```

Comment:
on the app-data readme I see different constant values, which is the correct one? https://github.com/cowprotocol/app-data#constants 

```js
import {
  LATEST_APP_DATA_VERSION,
  LATEST_QUOTE_METADATA_VERSION,
  LATEST_REFERRER_METADATA_VERSION
} from '@cowprotocol/app-data'
```

Reply:
Those are constants for the versions "names": 0.1.0, etc
What is imported here are the version types.